docs/getting_started/install_using_ddev.md

```diff
@@ -0,0 +1,410 @@
+---
+description: Install Ibexa DXP with Docker and DDEV to use it for development.
+---
+
+# Install using DDEV
+
+This guide provides a step-by-step walkthrough of installing [[= product_name =]] using [DDEV](https://ddev.com/).
+DDEV is an open-source tool that simplifies the process of setting up local PHP development environments.
+
+## Requirements
+
+Before you start the installation, ensure you have the following software installed:
+
+- [Docker](https://docs.docker.com/get-docker/)
+- [DDEV](https://ddev.readthedocs.io/en/latest/users/install/ddev-installation/)
+
+## Installation
+
+### 1. Create a DDEV project directory
+
+Start by creating a directory for your DDEV project by using the following command to create a new directory:
+
+```bash
+mkdir my-ddev-project && cd my-ddev-project
+```
+
+Replace `my-ddev-project` with your desired directory name.
```

Comment:
Done in a75667c07f3b82d736f15ed0ee199a1a7dee77c3

Couldn't be applied directly from Github because of triple backquotes conflict between code and suggestion.